lib/handler/chunked.c

```diff
@@ -78,22 +78,22 @@ static void on_setup_ostream(h2o_filter_t *self, h2o_req_t *req, h2o_ostream_t *
     if (req->is_subrequest)
         goto Next;
```

Comment:
Shouldn't we jump to `NextWithoutTrailer`? Unless we do, I believe that there's a chance where `send_server_timing_trailer` remains true after returning from this function, when `req->is_subrequest` is true.

Reply:
Yes, but subrequests are never handled by protocol handlers, so any trailers won't be sent.